src/node_jsvmapi.h

```diff
@@ -116,80 +116,99 @@ enum napi_valuetype {
   napi_function,
 };
 
-// Environment
-NODE_EXTERN napi_env napi_get_current_env();
+enum napi_errorvalue {
+  napi_ok,
+  napi_invalid_arg,
+  napi_object_expected,
+  napi_string_expected,
+  napi_generic_failure
+};
 
+struct napi_extended_error_info {
+  const char* error_message;
+  void* engine_reserved;
```

Comment:
I think void* is right for now until we find a real way for an engine to use it. And, yeah, we definitely at least need napi_clear_extended_error_info.